### PR TITLE
Feature w2.7 review of survey results

### PR DESCRIPTION
**Pull request by jelenazeljko55**
I create a front-end and back-end for the survey results.

( System administrator can observe survey results (grades for different items). They can also observe grades and average grade for each doctor.)

### CODE REVIEW

**Comment by jelenazeljko55** [comment]
[(3) W2.7 As a system administrator I want to observe survey results so that I could find out if patients are satisfied with work of our hospital.](https://trello.com/c/SXJ0hqiQ/26-3-w27-as-a-system-administrator-i-want-to-observe-survey-results-so-that-i-could-find-out-if-patients-are-satisfied-with-work-of)

File: HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs [hunk 1]
```diff
@@ -18,24 +18,147 @@ public MySqlSurveyRepository(MyDbContext context)
...
         {
-            _context.Surveys.Add(survey);
+            _context.SurveysAboutDoctor.Add(survey.SurveyAboutDoctor);
+            _context.SurveysAboutMedicalStaff.Add(survey.SurveyAboutMedicalStaff);
+            _context.SurveysAboutHospital.Add(survey.SurveyAboutHospital);
             _context.SaveChanges();
         }
 
-        public Survey GetSurveyById(int id)
+        public List<SurveyResult> GetSurveyResultsAboutDoctor(string jmbg)
```
Comment (Pufke) [inline HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs:24]: A method contains too many lines of code. Generally, any method longer than ten lines should make you start asking questions.  

If you feel the need to comment on something inside a method, you should take this code and put it in a new method. Even a single line can and should be split off into a separate method, if it requires explanations. And if the method has a descriptive name, nobody will need to look at the code to see what it does.

To reduce the length of a method body, use Extract Method ( https://refactoring.guru/extract-method ).

File: HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs [hunk 2]
```diff
@@ -18,24 +18,147 @@ public MySqlSurveyRepository(MyDbContext context)
...
+            availabilityOfDoctor.NumberOfGradesThree = GetSurveysByDoctor(jmbg).Count(x => x.AvailabilityOfDoctor == 3);
+            availabilityOfDoctor.NumberOfGradesFour = GetSurveysByDoctor(jmbg).Count(x => x.AvailabilityOfDoctor == 4);
+            availabilityOfDoctor.NumberOfGradesFive = GetSurveysByDoctor(jmbg).Count(x => x.AvailabilityOfDoctor == 5);
+            surveyResults.Add(availabilityOfDoctor);
+
+            return surveyResults;
         }
 
-        public List<Survey> GetSurveysByJmbg(string jmbg)
+        public List<SurveyResult> GetSurveyResultsAboutHospital()
```
Comment (Pufke) [inline HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs]: same problem

File: HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs [hunk 3]
```diff
@@ -18,24 +18,147 @@ public MySqlSurveyRepository(MyDbContext context)
...
+            satisfiedWithDrugAndInstrument.NumberOfGradesThree = _context.SurveysAboutHospital.Count(x => x.SatisfiedWithDrugAndInstrument == 3);
+            satisfiedWithDrugAndInstrument.NumberOfGradesFour = _context.SurveysAboutHospital.Count(x => x.SatisfiedWithDrugAndInstrument == 4);
+            satisfiedWithDrugAndInstrument.NumberOfGradesFive = _context.SurveysAboutHospital.Count(x => x.SatisfiedWithDrugAndInstrument == 5);
+            surveyResults.Add(satisfiedWithDrugAndInstrument);
+
+            return surveyResults;
         }
 
-        public void UpdateSurvey(Survey survey)
+        public List<SurveyResult> GetSurveyResultsAboutMedicalStaff()
```
Comment (Pufke) [inline HealthcareSystem/Backend/Repository/SurveyRepository/MySqlSurveyRepository/MySqlSurveyRepository.cs]: same problem

**Review by Pufke — CHANGES_REQUESTED** [review]

**Review by Pufke — DISMISSED** [review]

**Review by vlaksi — DISMISSED** [review]
lgtm

**Review by milana-todorovic — COMMENTED** [review]
@jelenazeljko55 @borislav-puzic @PSW-2020-ORG4/devops 
Please check that build works for you and neither WebPatientApp or IntegrationAdapters projects lost anything, since the csproj had to be merged manually.

**Review by milana-todorovic — APPROVED** [review]
Seems like everything is fine.

**Review by anciz — APPROVED** [review]

**Pull request merged**